### ISSUE TITLE
multicov header

### ISSUE THREAD

**Issue by jacquelynNR** [arq5x/bedtools2#966]
I am using multicov to calculate coverages across specific regions for many bam files and would like to include a header for the output that includes the filename of the bam file that the coverage came from. I would greatly appreciate any help in figuring out how to complete this. Thank you!

An example of what I want is:
Chr   Start     Stop     bam1.bam   bam2.bam   bam3.bam
1       12322   12553   234              467              532

where 234 is the coverage of bam1.bam for chr1:12322-12553
and 467 is the coverage of bam2.bam for chr1:12322-12553

I am currently using the following command 
bedtools multicov -bams *.bam -bed P1.bed > cov.txt

How do I modify the above command to include the header with the BAM file names? Thank you!
Jacquelyn

**Comment by arq5x** [comment]
Yes, this tool is indeed missing a header option. I apologize - we will try to get this out in a next release.
